### PR TITLE
#120: API-core-parallel_dispatch-ParallelForTag from md to rst

### PR DESCRIPTION
**Pull request by antoinemeyer5**
### THIS PR:
* API/core/parallel-dispatch/ParallelForTag transition from .md to .rst

before (.md) | after (.rst) | after 2 (.rst) [current code]
:----------------: | :-------------: | :-------------: 
before_1 | after_1 | image

### CODE REVIEW

File: docs/source/API/core/parallel-dispatch/ParallelForTag.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+.. role::cpp(code)
+    :language: cpp
+
+Header File: ``<Kokkos_ExecPolicy.hpp>``
+
+.. _text: ../parallel-dispatch/parallel_for.html
+
+.. |text| replace:: ``parallel_for``
+
+A tag used in team size calculation functions to indicate that the functor for which a team size is being requested is being used in a |text|_
```
Comment (nmm0) [inline docs/source/API/core/parallel-dispatch/ParallelForTag.rst:13]: Same as before, probably easier to cross reference https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#role-cpp-func

**Review by nmm0 — APPROVED** [review]
Looks good to me!

**Comment by fnrizzi** [comment]
@antoinemeyer5 please address the link comment

File: docs/source/API/core/parallel-dispatch/ParallelForTag.rst (hunk 1 above)
Comment (antoinemeyer5) [reply docs/source/API/core/parallel-dispatch/ParallelForTag.rst]: done

**Pull request merged**